src/Controllers/UserController.cs

```diff
@@ -42,9 +41,76 @@ public ActionResult<UserModel> GetAllUsers()
         var users = collection.Find(_ => true).ToList();
         if (users.Count == 0)
         {
-            return NotFound(new { Message = $"No users found"});
+            return NotFound(new { Message = $"No users found" });
         }
 
         return Ok(users);
     }
+
+    [HttpPost]
+    public IActionResult AddUser([FromBody] UserModel user)
```

Comment:
what is FromBody? and does this require that UserModel and our User struct in Rust have the exact same properties?

Reply:
[FromBody] means that its getting the information from the body of the request rather than having it be entered in as a parameter in the url. For example it makes it where we dont require the url to look like `http://localhost:3000/api/user?id=1111`